crates/http/src/lib.rs

```diff
@@ -63,11 +63,11 @@ pub struct CliArgs {
     pub address: SocketAddr,
 
     /// The path to the certificate to use for https, if this is not set, normal http will be used. The cert should be in PEM format
-    #[clap(long, env = "SPIN_TLS_CERT", requires = "tls-key")]
+    #[clap(long = "tls-cert", env = "SPIN_TLS_CERT", requires = "tls_key")]
```

Comment:
This is a tricky change: clap v4 doesn't make arguments kebab case from snake case ids anymore.

Tests are passing locally and I did some searching, but I'm not sure this isn't causing a bug or breaking change somewhere.